src/business-logic/services/interfaces/rundown-event-emitter.ts

```diff
@@ -21,8 +21,10 @@ export interface RundownEventEmitter {
   emitSegmentCreated(rundown: Rundown, segment: Segment): void
   emitSegmentUpdated(rundown: Rundown, segment: Segment): void
   emitSegmentDeleted(rundown: Rundown, segmentId: string): void
+  emitSegmentUnsynced(rundown: Rundown, unsyncedSegment: Segment, originalSegmentId: string): void
 
   emitPartCreated(rundown: Rundown, part: Part): void
   emitPartUpdated(rundown: Rundown, part: Part): void
-  emitPartDeleted(rundown: Rundown, partId: string): void
+  emitPartDeleted(rundown: Rundown, segmentId: string, partId: string): void
+  emitPartUnsynced(rundown: Rundown, part: Part): void
```

Comment:
It's a bit weird that the parameters for `emitSegmentUnsynced` and `emitPartUnsynced` don't follow the same "pattern" since the names of the methods are so similar. 
It would be nice if there was some familiarity between the two to avoid potential confusion.

Reply:
I understand the pattern is off when you look here. 
The issue is the fact that the flow is not the same for segments and parts. Segment are emitted with their unsynced ID. Parts are not. 
I could try and change the flow to fit this if you want me to?

Reply:
I don't think you need to spend much time on it for now.